docs/source/markdown/podman-container-runlabel.1.md

```diff
@@ -1,111 +1,91 @@
 
 #### **--creds**=*[username[:password]]*
 
-The [username[:password]] to use to authenticate with the registry if required.
-If one or both values are not supplied, a command line prompt will appear and the
-value can be entered.  The password is entered without echo.
+The [username[:password]] to use to authenticate with the registry if required.  If one or both values are not supplied, a command line prompt will appear and the value can be entered.  The password is entered without echo.
 
 #### **--help**, **-h**
 Print usage statement
 
 #### **--name**, **-n**=*name*
 
-Use this name for creating content for the container. NAME will default to the IMAGENAME if it is not specified.
+Use this name for creating content for the container.  If not specified, name defaults to the name of the image.
 
 #### **--quiet**, **-q**
 
 Suppress output information when pulling images
 
 #### **--replace**
 
-If a container exists of the default or given name, as needed it will be stopped, deleted and a new container will be
-created from this image.
+If a container exists of the default or given name, as needed it will be stopped, deleted and a new container will be created from this image.
 
 #### **--tls-verify**
 
-Require HTTPS and verify certificates when contacting registries (default: true). If explicitly set to true,
-then TLS verification will be used. If set to false, then TLS verification will not be used. If not specified,
-TLS verification will be used unless the target registry is listed as an insecure registry in registries.conf.
+Require HTTPS and verify certificates when contacting registries (default: true). If explicitly set to true, then TLS verification will be used. If set to false, then TLS verification will not be used. If not specified, TLS verification will be used unless the target registry is listed as an insecure registry in containers-registries.conf(5).
 
 ## EXAMPLES
 
-Execute the run label of an image called foobar.
+Execute the `run` label of an image called foobar.
 ```
-$ sudo podman container runlabel run foobar
+$ podman container runlabel run foobar
 ```
 
-Execute the install label of an image called foobar with additional arguments.
+Execute the `install` label of an image called foobar with additional arguments.
 ```
-$ sudo podman container runlabel install foobar apples oranges
+$ podman container runlabel install foobar apples oranges
 ```
 
-Display the command that would be executed by runlabel.
+Display the contents of the `run` label of image foobar.
 ```
-$ sudo podman container runlabel --display run foobar
+$ podman container runlabel --display run foobar
 ```
 
 ## SEE ALSO
-podman(1), containers-certs.d(5)
+podman(1), crun(1), runc(1), containers-auth.json(5), containers-certs.d(5), containers-registries.conf(5)
 
 ## HISTORY
+August 2021, Refinements by Valentin Rothberg (rothberg at redhat dot com)
```

Comment:
You need to add an empty newline below, otherwise markdown renders this on the same line.

Reply:
Thanks!